deploy/templates/nstemplatetiers/test/cluster.yaml

```diff
@@ -6,15 +6,38 @@ objects:
 - apiVersion: quota.openshift.io/v1
   kind: ClusterResourceQuota
   metadata:
-    name: for-${USERNAME}-compute
+    name: for-${USERNAME}-compute-deploy
   spec:
     quota:
+      scopes:
+        - NotTerminating
```

Comment:
AFAIK scopes are supported in `ResoruceQuotas` only. They are not supported in `ClusterResourceQuotas`

Reply:
This is why we moved the compute and storage quotas to the `ns_*.yaml` from the `cluster.yaml` in our AppStudio tier: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml#L57-L93

Reply:
Thanks.  Will do.